tests/symbolic/option-no-match.stuck.tzt

```diff
@@ -0,0 +1,15 @@
+input { Stack_elt nat $I1 } ;   //        I1
+code { DUP                      //     I1 I1
+     ; PUSH nat 0               //   0 I1 I1
+     ; COMPARE                  //   ???? I1
+     ; LT                       //   0>I1 I1
+     ; IF { SOME            }   // (Some I1) where I1 > 0
+          { DROP ; NONE nat }   //      None
+     ; PUSH nat 2
```

Comment:
Can you add a comment that the `PUSH nat 2` causes the stack shapes to not match?

Reply:
```suggestion
     ; PUSH nat 2                        // Have nat, expecting option
```